docs/wiki/EL/JSON-RPC.md

```diff
@@ -0,0 +1,113 @@
+| eth_call                 |  transaction object   | executes a new message call immediately without creating a transaction on the block chain |
+| eth_chainId              |  no mandatory params  | returns the current chain id                                     |
+| eth_estimateGas          |  transaction object   | makes a call or transaction, which won't be added to the blockchain and returns the used gas, which can be used for estimating the used gas |
+| eth_gasPrice             |  no mandatory params  | returns the current price per gas in wei                         |
+| eth_getBalance           |  address, block number | returns the balance of the account of the given address          |
+| eth_getBlockByHash       |  block hash, full txs  | returns information about a block by hash                        |
+| eth_getBlockByNumber     |  block number, full txs | returns information about a block by block number                |
+| eth_getBlockTransactionCountByHash | block hash | returns the number of transactions in a block from a block matching the given block hash |
+| eth_getBlockTransactionCountByNumber | block number | returns the number of transactions in a block from a block matching the given block number |
+| eth_getCode              |  address, block number | returns code at a given address in the blockchain                 |
+| eth_getLogs              |  filter object         | returns an array of all logs matching a given filter object       |
+| eth_getStorageAt         |  address, position, block number | returns the value from a storage position at a given address |
+
+## Tooling
+
+There are several ways of how to use the JSON-RPC methods. One of them is using the `curl` command. For example, to get the latest block number, you can use the following command:
+
+```bash
+curl <node-endpoint> \
+-X POST \
+-H "Content-Type: application/json" \
+-d '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}'
+```
+Please note how the *params* field is empty, as the method pass "latest" as default value.
+
+Another way is to use the `axios` library in Javascript/TypeScript. For example, to get the address balance, you can use the following code:
+
+```typescript
+import axios from 'axios';
+
+const node = '<node-endpoint>';
+const address = '<address>';
+
+const response = await axios.post(node, {
+  jsonrpc: '2.0',
+  method: 'eth_getBalance',
+  params: [address, 'latest'],
+  id: 1,
+  headers: {
+    'Content-Type': 'application/json',
+  },
+});
+```
+As you may notice, the JSON-RPC methods are wrapped in a POST request, and the parameters are passed in the body of the request.
+This is a different way to exchange data between the client and the server using the OSI's application layer: the HTTP protocol.
+
+Usually, all the web3 libraries wrap the JSON-RPC methods providing a more friendly way to interact with the execution layer. Please, look forward in your preferred programming language as the syntax may vary.
```

Comment:
This is a good point and apart of curl/axios, you should add an example from a library because that's most likely to be used anyway. E.g. web3.py makes it very simple https://web3py.readthedocs.io/en/stable/overview.html#overview
Regarding the axios example, I guess it's nice to have a generic js approach but most of users will go directly with ethers, web3.js or similar libraries